packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -24,12 +24,14 @@ import {
   ComponentStatus,
   RequestStatus
 } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import { SourceTracking } from '@salesforce/source-tracking';
```

Comment:
Do we already import @salesforce/source-tracking to the core module? I took a gander and don't think we do. 
Two thing should probably happen here. 
1. we should just export SourceTracking from the SourceTrackingService to isolate the dependency into utils. 
2. We should add @salesforce/source-tracking to the packageUpdates.dependancies in package.json for core so if something does fail there we don't get the message blow up error